spec.example.yaml

```diff
@@ -38,24 +38,44 @@ loadGenerator:
     - name: v1-5-2
     # Groups of queries that are launched independently.
     queryGroups:
-    - intervalSeconds: 4
+    - name: FirstGroup
+      intervalSeconds: 4
       queries:
-      - go_goroutines
-      - container_memory_rss
-      - kube_pod_container_info
-      - codelab_api_http_requests_in_progress
-      - codelab_api_requests_total
-    - intervalSeconds: 10
+      - type: query_range
+        queryString: go_goroutines
+        start: 1489165548
+        end: 1489165569
```

Comment:
The time range should be relative to `now`.
Fixed timestamps cause us to always query the same data. We also have to adjust them in every new deployment. More realistic would be to query a window like `[rel_end-dur:rel_end]`, where `rel_end` is a duration delta to now and `dur` the total window.